compiler/circle-eval-diff/driver/Driver.cpp

```diff
@@ -149,7 +150,12 @@ int entry(const int argc, char **argv)
 
   ced.init();
 
-  ced.evalDiff(first_input_data_path, second_input_data_path);
+  auto first_input_loader = circle_eval_diff::makeDataLoader(
+    first_input_data_path, to_input_format(input_data_format), ced.first_module_inputs());
+  auto second_input_loader = circle_eval_diff::makeDataLoader(
+    second_input_data_path, to_input_format(input_data_format), ced.second_module_inputs());
+
+  ced.evalDiff(first_input_loader.get(), second_input_loader.get());
```

Comment:
It seems that this change makes `CircleEvalDiff` have complex interface.

How about hiding the details (`makeDataLoader`) inside `CircleEvalDiff`? That's why I made `CircleEvalDiff::Context` have `input_format`.